### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vbevziuk**
- [DEMO LINK](https://vbevziuk.github.io/layout_calendar/)
- [TEST REPORT LINK](https://vbevziuk.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+@import "utils/day";
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
```
Comment (maxim2310) [inline src/styles/main.scss]: would be nice move it to reset file

File: src/index.html [hunk 2]
```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar calendar--start-day--thu calendar--month-length--29">
```
Comment (maxim2310) [inline src/index.html]:
```suggestion
    <div class="calendar calendar--start-day--thu calendar--month-length--29">
```

File: src/index.html [hunk 3]
```diff
@@ -7,6 +7,39 @@
...
+      <div class="day"></div>
```
Comment (maxim2310) [inline src/index.html]:
```suggestion
      <div class="calendar__day"></div>
```

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 10px;
```
Comment (maxim2310) [inline src/styles/main.scss]: block must havent external margin

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,34 @@
...
+  padding: 10px;
+  gap: 1px;
+  width: (($squeare-size*7)+6);
```
Comment (maxim2310) [inline src/styles/main.scss]:
```suggestion
  width: (($squeare-size * 7) + 6);
```

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,34 @@
...
+
+  @each $day, $number in $week {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($squeare-size+1) * $number;
```
Comment (maxim2310) [inline src/styles/main.scss]:
```suggestion
      margin-left: ($squeare-size + 1) * $number;
```

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple things needs to be fixed before approval)
and would be nice to center your calendar on page

File: src/styles/main.scss [hunk 7]
```diff
@@ -1,3 +1,57 @@
...
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```
Comment (maxim2310) [inline src/styles/main.scss]: would be better to center it using flex box to body

**Review by maxim2310 — APPROVED** [review]
Good job. Suggest some improvements.

File: src/styles/main.scss (hunk 7 above)
Comment (vbevziuk) [reply src/styles/main.scss]: > would be better to center it using flex box to body

why can't I center this element vertically with `align-items: center;`?

body {
  display: flex;
  justify-content: center;
  align-items: center;
  margin: 0;
}

**Pull request closed without merging**